### ISSUE TITLE
Tried to build this code on Visual Studio, lots of compile problems, any suggestions?

### ISSUE THREAD

**Issue by JacquesRoth** [tgautam03/tGeMM#1]
a_frag, b_frag, and c_frag declarations give undefined errors, adding "using namespace nvcuda;" did not help.  Adding "include <mma.h> did not help.  Is there something that must be configured in Visual Studio to enable tensor core operations?
Invocation of naive_tensor_mat_mul_kernel<<<dim_grid... gives expected an expression error.
"int warpM = blockIdx.x;" gives blockIdx undefined error!

Quite a few other errors.  Has anyone tried this code with Visual Studio for Cuda?

**Comment by tgautam03** [comment]
It could be a driver issue (assuming your GPU has tensor cores). Did you compile it using the make file I provided?

Try compiling using the following commands:

`make 00_benchmark_cuBLAS.out`

`make 01_benchmark_naive.out`

**Comment by JacquesRoth** [comment]
No, I am using Visual Studio and I don't know if there is some way to run your make file.  I am having the same issues with other code that invokes the tensor core functionality, just will not build without many errors.

**Comment by tgautam03** [comment]
I left two commands in the previous comment. Please try them out in the terminal (ensuring that the terminal is opened in tgemm folder) and see if that works.

**Comment by JacquesRoth** [comment]
These problems were due to issues with Visual Studio 2022 and Windows.  For some reason yet unknown the compilation of code with the nvcuda namespace does not work from the IDE Build command or Powershell but works from the Windows command prompt.
